### PR TITLE
Fix #152: HHH90000025: PostgreSQLDialect does not need to be specified explicitly

### LINKED ISSUE #152: HHH90000025: PostgreSQLDialect does not need to be specified explicitly
Fix following warning:

```
2023-11-02T10:57:09.340+01:00  WARN 56769 --- [           main] org.hibernate.orm.deprecation            : HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
```

### PR DESCRIPTION
**Pull request by jnpsk**

### CODE REVIEW

File: deploy/conf/application.properties [hunk 1]
```diff
@@ -27,7 +27,7 @@ spring.datasource.driverClassName=${MOBILE_UTILITY_SERVER_DATASOURCE_DRIVER:org.
 spring.jpa.properties.hibernate.connection.CharSet=${MOBILE_UTILITY_SERVER_JPA_CHARSET:}
 spring.jpa.properties.hibernate.connection.characterEncoding=${MOBILE_UTILITY_SERVER_JPA_CHARACTER_ENCODING:utf8}
 spring.jpa.properties.hibernate.connection.useUnicode=${MOBILE_UTILITY_SERVER_JPA_USE_UNICODE:true}
-spring.jpa.database-platform=${MOBILE_UTILITY_SERVER_JPA_DATABASE_PLATFORM:org.hibernate.dialect.PostgreSQLDialect}
+spring.jpa.database-platform=${MOBILE_UTILITY_SERVER_JPA_DATABASE_PLATFORM:}
```
Comment (banterCZ) [inline deploy/conf/application.properties:30]: I suggest removing this property completely. @romanstrobl @zcgandcomp Do you want to keep it?

Comment (korbelm) [reply deploy/conf/application.properties:30]: how about for other DB's like MSSQL? is it not needed as well?

Comment (banterCZ) [reply deploy/conf/application.properties:30]: https://vladmihalcea.com/hibernate-dialect/

Comment (zcgandcomp) [reply deploy/conf/application.properties:30]: I have no problem with removing the property completely - if hibernate will work correctly. But as of now the test will fail to run due to a missing `spring.jpa.database-platform`. So fix this first.

**Review by banterCZ — APPROVED** [review]
Looks good.

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Review by zcgandcomp — APPROVED** [review]
Seems OK

**Pull request merged**